.buildkite/heartbeat/heartbeat-pipeline.yml

```diff
@@ -197,12 +167,31 @@ steps:
           - "heartbeat/build/*.xml"
           - "heartbeat/build/*.json"
 
-  - group: "Packaging"
-    key: "packaging"
+  - group: "Heartbeat Packaging"
+    key: "heartbeat-packaging"
     if: build.env("BUILDKITE_PULL_REQUEST") != "false"
     depends_on:
-      - "mandatory-tests"
-
+      - "heartbeat-mandatory-tests"
     steps:
-      - label: Package pipeline
-        commands: ".buildkite/scripts/packaging/package-step.sh"
+      - label: ":ubuntu: Packaging Linux X86"
+        key: "heartbeat-package-linux-x86"
+        env:
+          PLATFORMS: "+all linux/amd64 linux/arm64 windows/amd64 darwin/amd64 darwin/arm64"
+          SNAPSHOT: true
+        command: ".buildkite/scripts/packaging/package.sh"
+        agents:
+          provider: gcp
+          image: "${IMAGE_UBUNTU_X86_64}"
+          machineType: "${GCP_HI_PERF_MACHINE_TYPE}"
+
+      - label: ":linux: Packaging Linux ARM"
+        key: "heartbeat-package-linux-arm"
+        env:
+          PLATFORMS: "linux/arm64"
+          PACKAGES: "docker"
+          SNAPSHOT: true
+        command: ".buildkite/scripts/packaging/package.sh"
```

Comment:
Since we are using our image, let's remove the usage of the wrapper script and replace this with `mage package, same applies to line 181

Reply:
I see we are still using `package.sh` in [main](https://github.com/elastic/beats/blob/main/.buildkite/scripts/packaging/package-step.sh#L23). I propose to avoid serious functional changes within this PR